index.html

```diff
@@ -2172,6 +2172,16 @@ <h2>Method Schemes</h2>
 formats as possible.
       </p>
 
+      <p>
+The <code>method-specific-id</code> format MAY include colons, which may be
```

Comment:
```suggestion
The <code>method-specific-id</code> format MAY include colons, which might be
```

Avoid using lowercase normative words... because people always complain about whether or not the document meant it to be a normative statement.